charts/opentelemetry-collector/Chart.yaml

```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: dmitryax
   - name: TylerHelmuth
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: 0.55.0
+appVersion: 0.56.0
```

Comment:
Please don't update the image version. It's an unrelated change

Reply:
Sorry about that, I was not really sure about what version should I update according to the contributing guide.